extension/chrome/settings/modules/auth_denied.htm

```diff
@@ -10,8 +30,14 @@ <h1>Explaining FlowCrypt webmail permissions</h1>
 </div>
 <div class="line">
   Nobody, FlowCrypt developers included, is able to access these permissions, they are stored privately in your
-  browser. Learn more about <a href="#" data-swal-page="/chrome/texts/email_provider_auth_security.htm">email
+  browser. Learn more about <a href="/chrome/texts/email_provider_auth_security.htm">email
     provider access and security</a>.
 </div>
-<div class="line">Please write human@flowcrypt.com if you have any questions or concerns.</div>
-<div class="line">Would you like to proceed to the permissions screen again?</div>
\ No newline at end of file
+<div class="line ask_support_assistance_container">Please write human@flowcrypt.com if you have any questions or concerns.</div>
```

Comment:
If it's being filled with javascript, then the tag should be empty - it will be less confusing that way.

Reply:
understand sir... thank you